### DISCUSSION TITLE
Ray Tracing Transmission Power

### DISCUSSION THREAD

**Discussion by TwiggyDaniels** [NVlabs/sionna#165] [Q&A]
I am hoping to generate coverage maps with ray tracing at different transmission powers to demonstrate how they change. I just found it strange that this is not a part of where we define a `Transmitter`, is it instead handled within where we define a `PlanarArray` with the `pattern` field?

I'm having a difficult time crawling through the source to make sense of things and my EM and 3GPP spec knowledge is not the best.

**Comment by faycalaa** [comment]
The coverage map provides for every point on it the average signal attenuation. This indicates how much a signal transmitted with an arbitrary power weakens upon reaching a receiver located at this point. The transmission power is a scale factor applied to this map.

**Reply by PhilSoy** [reply]
I did not find the way to set (and even to know) the transmitted power. i need to know the absolute transmit power for my application, i expected such parameter to be in 'transmitter' or in 'PlanarArray' but it is not.
How to proceed then for link level simulations ?

**Reply by jhoydis** [reply]
Hi @PhilSoy,

We never define an absolute transmit power (which is implicitly assumed to be equal to one) but rather scale the noise variance to achieve a desired SNR.
